### DISCUSSION TITLE
Material Request Pretty Please!

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#144] [Discuss a Topic]
Hey Dr. T. ,

I fell behind this morning trying to type and follow the material, at some point, I just was following along. Is it possible to get a code reference on GitHub of what we pair-programmed this morning, or should I just wait and examine the video tonight when it comes out? 

Thanks

**Comment by drteresavasquez** [comment]
There is always a ticket for class code alongs. Here is the one for this project: https://github.com/nss-evening-cohort-14/evening-client-side/discussions/140

**Reply by danicrosby** [reply]
Thank you, thank you, thank you!!!!!

**Reply by drteresavasquez** [reply]
Of course! I try to make sure everyone has the code needed so keep your eye on the class discussion topic and that should get you what you need. If you ever don't see something let me know and I'll hook it up.
